experiments/pages/client-side-aggregation/README.md

```diff
@@ -0,0 +1,21 @@
+# Client-side aggregate models
+
+Have you ever wanted to group and aggregate data from one or more models that didn't support server-side aggregation?
+
+Here's a Skuid Page that shows how you can do that.
+
+The fundamental approach is this:
+
+ 1. Create a Ui-Only Model to use as the "client-side Aggregate Model"
+ 1. Create a JavaScript Snippet which iterates over all rows in a source Model (of any Data Source), and groups the rows by a grouping field, and then aggregates other fields or concatenates their values as desired into fields on the Ui-Only Model.
+ 1. Add a Model Action on the source Model which runs this data grouping/aggregation snippet whenever that Model is queried/saved/cancelled -- whatever events you desire. If you re-run the snippet on change, you get live-updates to your aggregate model --- super cool!
+
+ ## What it looks like
+
+![Client-side data aggregation](screenshot.png)
+
+## How to use this
+
+1. Download the [XML](clientSideAggregation.xml) for this page.
+1. From the Skuid Pages list, click "Create new page", and select "Upload XML". Select the file you downloaded. The API Version for the page should automatically be detected as V2.
```

Comment:
- As a general style thing, we need to avoid quotation marks in docs and use bold or italics instead. This is not a necessary change here, but see if you like this.
- The bigger thing for this like is that not all versions of Skuid can auto-detect page version. Let's make this a little more agnostic:

```suggestion
1. From the Skuid Pages list, click **Create new page**, select **Upload XML**, and then select the file you downloaded. Newer Skuid releases will detect the page's API version, but if your release does not, be sure to select **v2** as the Skuid Page API version.
```